packages/@aws-cdk/aws-lambda-nodejs/README.md

```diff
@@ -17,6 +17,8 @@
 
 This library provides constructs for Node.js Lambda functions.
 
+⚠️ This module has a peer dependency on [`parcel-bundler`](https://www.npmjs.com/package/parcel-bundler) v1.x.
```

Comment:
I think this requires a bit more details. Something like this:

---

To use this module, you will need to add a dependency on `parcel-bundler` in your `package.json`:

```shell
yarn add parcel-bundler@^1.0.0
# or
npm install parcel-bundler@^1.0.0
```